pkg/distributor/distributor.go

```diff
@@ -19,29 +20,34 @@ import (
 )
 
 var (
-	sendDuration = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+	sendDuration = promauto.NewHistogramVec(prometheus.HistogramOpts{
 		Namespace: "loki",
 		Name:      "distributor_send_duration_seconds",
 		Help:      "Time spent sending a sample batch to multiple replicated ingesters.",
 		Buckets:   []float64{.001, .0025, .005, .01, .025, .05, .1, .25, .5, 1},
 	}, []string{"method", "status_code"})
-	ingesterAppends = prometheus.NewCounterVec(prometheus.CounterOpts{
+	ingesterAppends = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "loki",
 		Name:      "distributor_ingester_appends_total",
 		Help:      "The total number of batch appends sent to ingesters.",
 	}, []string{"ingester"})
-	ingesterAppendFailures = prometheus.NewCounterVec(prometheus.CounterOpts{
+	ingesterAppendFailures = promauto.NewCounterVec(prometheus.CounterOpts{
 		Namespace: "loki",
 		Name:      "distributor_ingester_append_failures_total",
 		Help:      "The total number of failed batch appends sent to ingesters.",
 	}, []string{"ingester"})
-)
 
-func init() {
-	prometheus.MustRegister(sendDuration)
-	prometheus.MustRegister(ingesterAppends)
-	prometheus.MustRegister(ingesterAppendFailures)
-}
+	bytesIngested = promauto.NewCounterVec(prometheus.CounterOpts{
+		Namespace: "loki",
+		Name:      "distributor_bytes_received_total",
+		Help:      "The total number of uncompressed bytes received per user",
+	}, []string{"org"})
```

Comment:
In cortex we used "user"[0]; in Loki we seem to be converging on the terms "instance" and "tenant".

[0] https://github.com/cortexproject/cortex/blob/master/pkg/distributor/distributor.go#L50

Reply:
Should I just run with tenant then? See: https://github.com/grafana/loki/issues/131

Reply:
Instance is preferred I'd say; tenant is used in docs/presentations, but thats about it.